docs/snippets/1password-push-secret.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: source-secret
+stringData:
+  source-key: "my-secret"
+---
+apiVersion: external-secrets.io/v1alpha1
+kind: PushSecret
+metadata:
+  name: pushsecret-example # Customisable
+spec:
+  deletionPolicy: Delete
+  refreshInterval: 1h
+  secretStoreRefs:
+    - name: 1password
+      kind: ClusterSecretStore
+  selector:
+    secret:
+      name: source-secret # Source Kubernetes secret
+  data:
+    - match:
+        secretKey: source-key # Source Kubernetes secret key to be pushed
+        remoteRef:
+          remoteKey: 1pw-secret-name # 1Password item/secret name
+          property: password         # (Optional) 1Password field type, default password
+      metadata:
+        vault: staging         # Optional the vault the secret is going to be pushed to, defaults to the first defined vault in the (Cluster)SecretStore
```

Comment:
We now provide versionable way of dealing with Metadata for providers described here: https://github.com/external-secrets/external-secrets/pull/3612/files#diff-b20a64fd8b46ed86e84612368fa70c6e59805645112f5f35d943b925cc43f3beR88-R97

And here is a sample implementation: 
The Spec definition
https://github.com/external-secrets/external-secrets/blob/main/pkg/provider/azure/keyvault/keyvault.go#L125

The usage.
https://github.com/external-secrets/external-secrets/blob/main/pkg/provider/azure/keyvault/keyvault.go#L573

Reply:
I migrated in [`b277932` (#4173)](https://github.com/external-secrets/external-secrets/pull/4173/commits/b2779324cd2cf83fcd9ae3ae5032700ae01ea9ac)
Hope this correct